R/vegawidget.R

```diff
@@ -64,14 +91,38 @@ vegawidget <- function(spec, embed = NULL, width = NULL, height = NULL,
   # autosize (if needed)
   spec <- vw_autosize(spec, width = width, height = height)
 
-  # use internal methods here because spec has already been validated
-  x <-
-    .as_json(
-      list(
-        chart_spec = .as_list(spec),
-        embed_options = embed
-      )
+  # if base_url is a local directory need to create a depencency
```

Comment:
Again, we can address this later, but we might consider the case where a `baseURL` has been specified using a `loader` in `vega_embed()`.